pkg/logs/client/http/destination.go

```diff
@@ -167,7 +171,7 @@ func CheckConnectivity(endpoint config.Endpoint) config.HTTPConnectivity {
 	ctx := client.NewDestinationsContext()
 	ctx.Start()
 	defer ctx.Stop()
-	destination := NewDestination(endpoint, JSONContentType, ctx)
+	destination := newDestination(endpoint, JSONContentType, ctx, time.Second*5)
```

Comment:
a small comment inline to explain why we use a lower timeout would be nice

Reply:
Comment added 😃